app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,28 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user, only: [:create, :destroy, :index]
+
+  def index
+    @followings = current_user.followings
+  end
+
+  def create
+    @relationship = current_user.relationships.create(following_id: params[:following_id])
+
+    if @relationship.errors.any?
+      @relationship.errors.messages.values.each do |msg|
+        flash[:error] = "#{msg.first}"
+        redirect_to :back
+      end
+    else
+      flash[:notice] = "You are now following #{User.find(@relationship.following_id).name}"
```

Comment:
You could have `User.find(@relationship.following_id)` this at line 9, before `@relationship = current_user ...` and save it to a variable 'leader', you could use that variable in `@relationship = current_user.relationships.create(leader: leader)` (i hope you get the idea) and then in this flash message you could access it as well `leader.name`.